pkg/auth/auth_handler.go

```diff
@@ -212,6 +217,10 @@ func (a *AuthHandler) CreateAuthenticator() func(name, in string, authenticate s
 		getToken := func(r *http.Request) string { return r.Header.Get(name) }
 
 		return security.HttpAuthenticator(func(r *http.Request) (bool, interface{}, error) {
+			now := time.Now()
+			defer func() {
+				a.log.Infof("HttpAuthenticator took %v", time.Since(now))
```

Comment:
Why we have it here and also in `AuthUserAuth` ?

Reply:
was not sure what was the best func, so added in 2 of them

Reply:
So we can have it only here, it will work for both agent and user auth.